src/test/java/moduletwo5/test/TestRunner.java

```diff
@@ -0,0 +1,131 @@
+package moduletwo5.test;
+
+import moduletwo5.page.PassportYandexPage;
+import moduletwo5.page.YandexDiskPage;
+import moduletwo5.page.YandexSignInPage;
+import moduletwo5.page.constants.StringСonstants;
+import moduletwo5.page.enums.MainMenu;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterClass;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+
+public class TestRunner {
+
+    private WebDriver driver;
+    private PassportYandexPage passportYandexPage;
+    private YandexDiskPage yandexDiskPage;
+
+    @BeforeClass
+    public void getPrecondition() {
+        System.setProperty("webdriver.chrome.driver", "src/main/resources/chromedriver.exe");
+        driver = new ChromeDriver();
+        setWaitingsForDriver(driver);
+        YandexSignInPage yandexSignIn = new YandexSignInPage(driver);
+        yandexSignIn.openPage();
+        passportYandexPage = yandexSignIn.signIn();
+    }
+
+    @Test(description = "Test login into yandex disk using credentials (negative checks)")
+    public void negativeCheckSignInYande() {
+        passportYandexPage.enterWrongLoginAndPasswd(StringСonstants.WRONG_LOGIN, StringСonstants.WRONG_PASSWD);
+        Assert.assertTrue(passportYandexPage.haveAnErrorMessage(),
+                "The username or password you entered is correct.");
+    }
+
+    @Test(description = "Test login into yandex disk using credentials (positive checks)",
+            dependsOnMethods = "negativeCheckSignInYande")
+    public void positiveCheckSignInYande() {
+        driver.navigate().back();
+        driver.navigate().refresh();
+        yandexDiskPage = passportYandexPage.enterValidLoginAndPasswd(StringСonstants.VALID_LOGIN, StringСonstants.VALID_PASSWD);
+        Assert.assertEquals(StringСonstants.VALID_LOGIN, yandexDiskPage.getAccountName(),
+                "The username or password you entered is incorrect.");
+    }
+
+    @Test(description = "Test that all main menu items works correctly and lead to correct page",
+            dependsOnMethods = "positiveCheckSignInYande")
+    public void checkMainMenu() {
```

Comment:
Consider using SoftAssert for multiple assertions. In case of errors, some lines of this method become unreachable.